ignore/src/dir.rs

```diff
@@ -75,7 +75,7 @@ struct IgnoreOptions {
 
 impl IgnoreOptions {
     /// Returns true if at least one type of ignore rules should be matched.
-    fn should_ignores(&self) -> bool {
+    fn has_any_ignore_options(&self) -> bool {
```

Comment:
As mentioned in the commit message, I'm not sure whether this or simply `any` is a better name.